content/Result_2.py

```
@@ -106,141 +107,119 @@ def remove_substrings(original_string, substrings_to_remove):
             if (selected_file.find("Example") != -1): 
-               file_name_wout_out = "Example_RNA_UV_XL"
+               file_name_wout_out = "Example"
             else: 
                 file_name_wout_out = selected_file.replace(".idXML", "")
 
-         
-            if selected_mzML_file: 
-                MS2 = process_mzML_file(os.path.join(Path.cwd().parent ,  str(st.session_state.workspace)[3:] , "mzML-files" ,selected_mzML_file))
-                if MS2 is None:
-                    st.warning("The corresponding " +  ".mzML file could not be found. Please re-upload the mzML file to visualize all peaks.")
+            MS2 = process_mzML_file(os.path.join(Path.cwd().parent ,  str(st.session_state.workspace)[3:] , "mzML-files" ,f"{file_name_wout_out}.mzML"))
```

Ruff (0.12.2)

`os` may be undefined, or defined from star imports

(F405)